docs/installation/linux.txt

```diff
@@ -46,7 +46,7 @@ Scroll further below for more OS-specific insructinos.
     cd bigartm
     mkdir build && cd build
     cmake ..
-    make
+    make -j20
```

Comment:
I realized that another point for plain `make` would be the fact that in case of problems during build the error output would be much more sane.

I was a bit upset by the fact that I needed to stop running build process and rerun it with higher parallelism. I expected that this codeblock will use the most suitable approach for me.